src/pudl/transform/ferc1.py

```diff
@@ -2670,7 +2638,7 @@ class TransmissionFerc1TableTransformer(Ferc1AbstractTableTransformer):
     has_unique_record_ids: bool = False
 
 
-class ElectricEnergyAccountSourcesFerc1TableTransformer(Ferc1AbstractTableTransformer):
+class ElectricEnergySourcesFerc1TableTransformer(Ferc1AbstractTableTransformer):
```

Comment:
Just standardized this to match the table name and the disposition table transformer.